hazelcast/src/main/java/com/hazelcast/map/impl/record/CachedDataRecordWithStats.java

```diff
@@ -89,4 +89,10 @@ public int hashCode() {
         return result;
     }
 
+    @Override
+    public String toString() {
+        return "CachedDataRecordWithStats{"
+                + "cachedValue=" + cachedValue
+                + "} " + super.toString();
```

Comment:
minor: probably add the `super.toString()` output within the `{ }` brackets, so it is clearer that it is also part of the same object? (same would apply to other `toString()` impls)

Current version looks like this: `CachedDataRecordWithStats{cachedValue=null} DataRecordWithStats{value=null} AbstractRecordWithStats{lastStoredTime=-1, expirationTime=-1} AbstractRecord{key=null, ttl=0, maxIdle=0, version=0, hits=0, lastAccessTime=-1, lastUpdateTime=-1, creationTime=-1, metadata=null}`

```suggestion
                + ", super=" + super.toString() + "}";
```

Reply:
done.